rpc_jobs/rpc_aio_leapfrog.yml

```diff
@@ -0,0 +1,93 @@
+- project:
+    name: 'RPC-AIO-Leapfrog-Testing'
+    # Note: branch is the branch for periodics to build
+    #       branches is the branch pattern to match for PR Jobs.
+    series:
+      - newton-r11.1.18:
+          UPGRADE_FROM_REF: "r11.1.18"
+          branch: "r14.2.0"
+          branches: "r14.2.0"
+          KIBANA_SELENIUM_BRANCH: "newton"
```

Comment:
Think we're about there.  Last question (I hope) that I have: do we need `DEPLOY_TELEGRAF: "yes"` like we have [here](https://github.com/rcbops/rpc-gating/blob/master/rpc_jobs/rpc_aio.yml#L21)?

Reply:
Yeah, went ahead and added them for consistency.